iron-selectable.html

```diff
@@ -336,10 +346,19 @@
       if (this.selectedAttribute) {
         this.toggleAttribute(this.selectedAttribute, isSelected, item);
       }
+      if (this.selectedPropertyValue) {
+        item[this.selectedAttribute] = this.selectedPropertyValue;
+      }
       this._selectionChange();
       this.fire('iron-' + (isSelected ? 'select' : 'deselect'), {item: item});
     },
 
+    _updatePropertyValue: function(selectedItem, selectedAttribute, propertyValue) {
+      if (selectedAttribute && propertyValue) {
+        selectedItem[selectedAttribute] = propertyValue;
```

Comment:
That would break subviews though wouldn't it? We would have `app-location` intercepting the new route which would be passed down through `subroute`, but then the view never gets notified of a page change because the view's `route` didn't get updated.